### PR TITLE
added show databases metrics of interest specifically for alerting

### PR DESCRIPTION
**Pull request by mbroers**
Looks ok to me in my local environment.

### CODE REVIEW

**Comment by mbroers** [comment]
Sure thing, standby.

File: prometheus_pgbouncer_exporter/collector.py [hunk 1]
```diff
@@ -80,6 +80,19 @@ def collect(self):
...
 
+            # SHOW DATABASES
+            results = self._fetchMetrics(conn, "SHOW DATABASES")
+            if results:
+                results = self._filterMetricsByIncludeDatabases(results, self.config.getIncludeDatabases())
+                results = self._filterMetricsByExcludeDatabases(results, self.config.getExcludeDatabases())
+                metrics += self._exportMetrics(results, "pgbouncer_databases_", [
+                    {"type": "gauge", "column": "pool_size",           "metric": "database_pool_size",          "help": "Configured Pool Size Limit"},
+                    {"type": "gauge", "column": "reserve_pool",        "metric": "database_reserve_pool",       "help": "Configured Reserve Limit"},
+                    {"type": "gauge", "column": "current_connections", "metric": "database_current_connection", "help": "Database connection count"},
```
Comment (pracucci) [inline prometheus_pgbouncer_exporter/collector.py]: I would suggest to rename the metric to `database_current_connections`

File: tests/test_collector.py [hunk 2]
```diff
@@ -21,6 +21,11 @@ def fetchMetricsSuccessMock(conn, query):
             {"database": "test", "user": "marco", "cl_active": 1, "cl_waiting": 2, "sv_active": 3, "sv_idle": 4, "sv_used": 5, "sv_tested": 6, "sv_login": 7, "maxwait": 8 },
             {"database": "prod", "user": "marco", "cl_active": 8, "cl_waiting": 7, "sv_active": 6, "sv_idle": 5, "sv_used": 4, "sv_tested": 3, "sv_login": 2, "maxwait": 1 }
         ]
+    elif query == "SHOW DATABASES":
```
Comment (pracucci) [inline tests/test_collector.py]: What's missing is a test on these fixtures. For example you could add a test like this:

```
    def testShouldExportDatabasesMetrics(self):
        config = PgbouncerConfig({})
        collector = PgbouncerMetricsCollector(config)
        collector._createConnection = MagicMock(return_value=False)
        collector._fetchMetrics = MagicMock(side_effect=fetchMetricsSuccessMock)

        metrics = getMetricsByName(collector.collect(), "pgbouncer_databases_database_pool_size")
        self.assertEqual(len(metrics), 2)
        self.assertEqual(metrics[0]["type"], "gauge")
        self.assertEqual(metrics[0]["value"], 50)
        self.assertEqual(metrics[0]["labels"], {"database":"testpool"})
        self.assertEqual(metrics[1]["type"], "gauge")
        self.assertEqual(metrics[1]["value"], 90)
        self.assertEqual(metrics[1]["labels"], {"database":"prodpool"})

        # TODO test reserve_pool
        # TODO test current_connections
```

**Comment by mbroers** [comment]
Thanks for the patient feedback.  Im not much of a python developer so it helps to be pointed in the right direction.  I will make adjustments asap.

**Comment by mbroers** [comment]
whoops disregard this for a moment..

**Comment by mbroers** [comment]
ok, sorry for the noise here you go

**Comment by mbroers** [comment]
Hi, just checking in if there is anything else needed to get this merged or if I missed something?

**Comment by pracucci** [comment]
@mbroers Super sorry for the very late merge. I'm going fix a couple of typos in the README and then I will cut a new version.

**Comment by pracucci** [comment]
@mbroers [v. 1.0.0](https://github.com/spreaker/prometheus-pgbouncer-exporter/releases/tag/1.0.0) has been cut. However, pay attention that I've renamed the `database` label to `backend_database` and the `name` label to `database` for the three new metrics in this PR, in order to keep consistency with the old metrics (so that you can match metrics by `database` label that now has the same meaning).

**Pull request merged**